### ISSUE TITLE
Focus on error message when an invalid test result is uploaded

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5134]
## Area or Component
Test results bulk upload

## Quick Summary
Error message that appears does not gain focus when an invalid csv is uploaded.

### How to Replicate
Upload an invalid results csv. An error message will appear but it will not gain focus.

## Accessibility Violation

[why is this inaccessible? include deque link if available.]: #

## Action Requested
Ensure the error message gains focus once it appears on the page.

## Source Code (optional)

[which file or line is the issue attached to?]: #

**Comment by emmastephenson** [comment]
This should also be checked in the patient bulk upload, since the UI is very similar.
